src/controller/python/BUILD.gn

```diff
@@ -27,43 +27,51 @@ if (current_os == "mac") {
 }
 
 config("controller_wno_deprecate") {
-  cflags = [ "-Wno-deprecated-declarations" ]
+  cflags = [ 
+    "-Wno-deprecated-declarations", 
+  "-Wno-shadow",
+  "-Wno-unused-result",
+  "-Wsign-compare",
+  "-Wunreachable-code"
+  ]
 }
 
-shared_library("ChipDeviceCtrl") {
-  output_name = "_ChipDeviceCtrl"
-  output_dir = "${target_out_dir}/chip"
-  include_dirs = [ "." ]
+shared_library("CHIPController") {
+  output_name = "CHIPController"
+  output_dir = "${target_out_dir}/PyCHIPController"
+  include_dirs = [ 
+                  "." , 
+                  "/usr/local/include/pybind11/", 
```

Comment:
You should call `python3 -m pybind11 --includes` to obtain these paths imo.

Reply:
Yes! Not too familiar with GN, so I am going to ask someone from Google to help out with making the compilation more dynamic.

Reply:
I've committed to assisting with this part.